presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7374,6 +7374,12 @@ public void testValuesWithUnusedColumns()
         assertEquals(actual.getMaterializedRows(), expected.getMaterializedRows());
     }
 
+    @Test
```

Comment:
No need for this to be in AbstractTestQueries, which is used to test all connectors. Create a separate test like com.facebook.presto.sql.query.TestJoinUsing